src/components/calcite-accordion-item/calcite-accordion-item.scss

```diff
@@ -83,14 +83,10 @@
     text-color-3
     duration-150
     ease-in-out;
-  margin-right: var(--calcite-accordion-item-icon-spacing-start);
-  margin-left: var(--calcite-accordion-item-icon-spacing-end);
+  margin-inline-end: var(--calcite-accordion-item-icon-spacing-start);
```

Comment:
I chose to do these like this because the variable says start/end but its being used for the opposite sometimes. I think this could be refactored more in the future.